.pre-commit-config.frontend.yaml.jinja

```
@@ -0,0 +1,72 @@
+{% set eslint_files = '\\.(js|ts|vue)$' %}
+{% set eslint_exclude = '^nuxt-template-overrides/.*$' %}
+
+{% extends '.pre-commit-config.yaml.jinja' %}
+
+{% block note %}
+{{ super() }}
+# - https://github.com/WordPress/openverse/blob/main/.pre-commit-config.frontend.yaml.jinja
+{%- endblock %}
+
+{#==================================
+Blocks must be indented by 2 spaces.
+==================================#}
```

@sarayourfriend I hope this is clearer?
```suggestion
{#=======================================
Repo blocks must be indented by 2 spaces.
=======================================#}
```